actionview/lib/action_view/base.rb

```diff
@@ -270,6 +270,8 @@ def _run(method, template, locals, buffer, &block)
       _old_output_buffer, _old_virtual_path, _old_template = @output_buffer, @virtual_path, @current_template
       @current_template = template
       @output_buffer = buffer
+      @lookup_context.current_template = template
```

Comment:
We still shouldn't have to do this. See @tenderlove's comment about `@current_template` in master: https://github.com/rails/rails/pull/22473#issuecomment-475051040

Reply:
@kaspth updated. Should be right this time.